### DISCUSSION TITLE
Hold "Shift" to bypass Delete Confirmations

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#18060] [Ideas] [answered]
Or any button in particular.

I'm only proposing this because mass deleting a lot of scores/maps/collections (without nuking the entire save) is really tedious and annoying with hold-to-confirm:

https://user-images.githubusercontent.com/64581009/166315725-3b16a174-1329-48b8-bcb5-ca5991794043.mp4

I have only 3 proposals for this issue:
1. Holding a certain button to bypass this completely (Shift)
2. Make the Hold-To-Confirm delay setting also apply to the confirmation
3. Keep it as it is. Maybe it's not that bad as I think it is

**Comment by frenzibyte** [comment, marked as answer]
You can bypass it by explicitly pressing the number key associated with the button's position in the dialog.

**Reply by NotGumballer91** [reply]
Not really intuitive as I hoped but it works out

**Comment by peppy** [comment]
A bit hidden, but for that particular scenario you may want to also use the "clear all scores" option:

![osu Game Tests 2022-05-03 at 06 06 33](https://user-images.githubusercontent.com/191335/166410851-854e1daf-00f3-42f5-a8e9-4053b35af512.png)
